2/Dockerfile.rhel7

```diff
@@ -42,7 +42,7 @@ RUN yum-config-manager --disable epel >/dev/null || : && \
     # will subsequently get redirected to /usr/lib/jenkins; it is confirmed that the 3.7 jenkins RHEL images do *NOT* have a /usr/lib64/jenkins path
     ln -s /usr/lib/jenkins /usr/lib64/jenkins && \
     x86_EXTRA_RPMS=$(if [ "$(uname -m)" == "x86_64" ]; then echo -n java-1.8.0-openjdk.i686 java-1.8.0-openjdk-devel.i686 ; fi) && \
-    INSTALL_PKGS="dejavu-sans-fonts wget rsync gettext git tar zip unzip openssl bzip2 dumb-init  java-1.8.0-openjdk atomic-openshift-clients jenkins-2.* jenkins-2-plugins" && \
+    INSTALL_PKGS="dejavu-sans-fonts wget rsync gettext git tar zip unzip openssl bzip2 dumb-init  java-1.8.0-openjdk java-1.8.0-openjdk-devel atomic-openshift-clients jenkins-2.* jenkins-2-plugins" && \
```

Comment:
Can you elaborate on how adding the devel package assists us here?

Reply:
You are installing it on  x86:
`x86_EXTRA_RPMS=$(if [ "$(uname -m)" == "x86_64" ]; then echo -n java-1.8.0-openjdk.i686 java-1.8.0-openjdk-devel.i686 ; fi) && \`

I added it to INSTALL_PKGS so it could be picked up by all archs..unless you just the i686 version only for x86?

Reply:
ah missed that ... thanks

Reply:
Actually.. this logic will install java-1.8.0-openjdk-devel.i686 and java-1.8.0-openjdk-devel.x86_64 on x86...I think that is still ok?

Reply:
yes we want both for x86